test/parallel/test-async-hooks-enable-recursive.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const common = require('../common');
+const async_hooks = require('async_hooks');
+const crypto = require('crypto');
```

Comment:
You kept `crypto` here, was that intentional? If so, it probably needs a guard for skipping the test if appropriate